scripts/create_local_env_for_cil_development.sh

```diff
@@ -68,6 +68,12 @@ conda_args=(create --name="$name"
   tqdm
   zenodo_get'>=1.6'
 )
+
+if [[ "$(uname)" == "Linux" ]]; then
+  conda_args+=(libgcc-ng
+  libstdcxx-ng)
```

Comment:
could be one-line since it's short...
```suggestion
  conda_args+=(libgcc-ng libstdcxx-ng)
```